e2e-tests/content-strings/fields/index.js

```diff
@@ -205,4 +205,8 @@ export const FIELDS = {
       TITLE: 'Standard industrial classification (SIC) codes and nature of business',
     },
   },
+  [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.YOUR_COMPANY.PHONE_NUMBER]: {
+    LABEL: 'Your UK telephone number (optional)',
+    HINT: 'We may need to contact you about your application',
+  },
```

Comment:
Not a blocker for this PR but we need to split this file up and have all insurance fields in the `INSURANCE` object

Reply:
will do in next PR